components/style/build_gecko.rs

```diff
@@ -558,6 +558,8 @@ mod bindings {
             "nsresult",
             "Loader",
             "ServoStyleSheet",
+            "EffectCompositor_CascadeLevel",
```

Comment:
This is impossible, because this array is manually kept (we just use `--raw-line` to add the includes). But whatever :)